docker-compose.yaml

```diff
@@ -152,15 +154,15 @@ services:
     volumes:
       - ./data/mlflow:/data
 
-  web_client:
-    build: web_client
-    environment:
-      - COGMENT_VERSE_GRPCWEBPROXY_PUBLIC_URL
-      - REACT_APP_GRPCWEBPROXY_URL=${COGMENT_VERSE_GRPCWEBPROXY_PUBLIC_URL}
-      - PORT=${COGMENT_VERSE_WEBCLIENT_PORT}
-    restart: on-failure
-    ports:
-      - ${COGMENT_VERSE_WEBCLIENT_EXPOSED_PORT}:${COGMENT_VERSE_WEBCLIENT_PORT}/tcp
+  # web_client:
```

Comment:
Why is the web_client service removed? Do you plan to revert this?

Reply:
I do plan on reverting this, however it seems that this hasn't worked for a while, I can put a fix in this PR if you would like

Reply:
I think it's an oversight

Reply:
done